File: .sdk/tm/php/utility/PrepareBody.php
<?php
declare(strict_types=1);

// Thesmsworks SDK utility: prepare_body

class ThesmsworksPrepareBody
{
    public static function call(ThesmsworksContext $ctx): mixed
    {
        if ($ctx->op->input === 'data') {
            return ($ctx->utility->transform_request)($ctx);
        }
        return null;
    }
}

File: java/utility/PrepareBody.java
package voxgig.thesmsworkssdk.utility;

import voxgig.thesmsworkssdk.core.Context;

final class PrepareBody {

  private PrepareBody() {}

  static Object prepareBody(Context ctx) {
    if ("data".equals(ctx.op.input)) {
      return ctx.utility.transformRequest.apply(ctx);
    }

    return null;
  }
}
